source/common/http/codec_client.cc

```diff
@@ -247,7 +247,6 @@ void CodecClient::ActiveRequest::decodeHeaders(ResponseHeaderMapPtr&& headers, b
                 .http3Options()
                 .override_stream_error_on_invalid_http_message()
                 .value())) {
-        parent_.host_->cluster().trafficStats()->upstream_cx_protocol_error_.inc();
```

Comment:
Where is the duplicate increment of this? Can you clarify that? (My naive grep "upstream_cx_protocol_error" didn't find it)

Reply:
It happens in the CodecClient::onData method after dispatching wire bytes. https://github.com/envoyproxy/envoy/blob/6b99feb17872146adeb4fc3d5a9c26b9ab4ef996/source/common/http/codec_client.cc#L176